rllib/core/testing/torch/bc_module.py

```diff
@@ -12,13 +13,28 @@
 torch, nn = try_import_torch()
 
 
+@dataclass
+class DiscreteBCTorchModuleConfig(RLModuleConfig):
+    observation_space: gym.Space = None
+    action_space: gym.Space = None
+    model_config: Mapping[str, Any] = None
+
+    def build(self, framework):
+        if framework != "torch":
+            raise ValueError("Only torch framework is supported")
+        DiscreteBCTorchModule(self)
```

Comment:
Had to add these configs here or else all this BC code does not make sense in the context of other changes since December.